pkg/util/ip_address.go

```diff
@@ -10,12 +11,15 @@ func ParseIPAddress(input string) string {
 	host, _ := SplitHostPort(input)
 
 	ip := net.ParseIP(host)
-
 	if ip == nil {
 		return host
 	}
 
 	if ip.IsLoopback() {
+		if strings.Contains(host, ":") {
```

Comment:
This change makes the session view look like this. The earlier loopback check and returning 127.0.0.1 was when signing in locally, not sure though what makes the best user experience here?

Top: With your changes, bottom: Current master
![image](https://user-images.githubusercontent.com/1668778/66071451-776a1d00-e553-11e9-933f-9f40f6050efd.png)

Reply:
Maybe that's more of a UX issue though and shouldn't matter how we store the IP address

Reply:
@marefr I thought this change should only cause `ParseIPAddress` to return "::1" for IPv6 loopback addresses? Is your local address IPv6? I don't see why your network IP address would contain colons if not IPv6.